ingest/src/main/java/com/microsoft/azure/kusto/ingest/QueuedIngestClientImpl.java

```diff
@@ -46,13 +30,15 @@
 import java.lang.invoke.MethodHandles;
 import java.net.URISyntaxException;
 import java.time.Instant;
-import java.util.*;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.UUID;
 
 public class QueuedIngestClientImpl extends IngestClientBase implements QueuedIngestClient {
 
+    public static final String QUEUED_INGEST_CLIENT_IMPL = "QueuedIngestClientImpl";
```

Comment:
We don't really need to do it like this, do we?

Reply:
Out of scope for this PR.

This name is for tracing, it's either that, doing reflection every time, or hardcoding the name in the function.

Reply:
Yeah, OOS, but too ugly for me to ignore :-)

`this.getClass().getSimpleName()` is cleaner.

Reply:
Done